app/Http/routes.php

```diff
@@ -12,6 +12,7 @@
     Route::get('conferences/{conference}', 'ConferencesController@show');
 });
 
+Route::post('api/conferences/{conference}/introduction', 'ConferenceIntroductionController@store');
```

Comment:
ahhhh I thought it was because you wanted to avoid having to have an API-namespaced conference introduction controller.

That makes sense!